content/terraform-plugin-framework/v1.15.x/docs/plugin/framework/resources/identity.mdx

```diff
@@ -0,0 +1,462 @@
+```go
+{
+  "account_id": {string, required_for_import:false},
+  "partition": {string, required_for_import:false},
+  "zone_id": {string, required_for_import:true},
+  "record_name": {string, required_for_import:true},
+  "record_type": {string, required_for_import:true},
+}
+```
+
+Example identity data:
+```go
+{
+  "account_id": "123456789012",
+  "partition": "aws",
+  "zone_id" = "Z4KAPRWWNC7JR",
+  "record_name" = "dev.example.com",
+  "record_type" = "NS"
+}
+```
+
+#### AzureRM
+
+##### azurerm_virtual_network
+
+Example schema:
+```go
+{
+  "subscription_id": {string, required_for_import:true},
+  "resource_group_name": {string, required_for_import:true},
+  "virtual_network_name": {string, required_for_import:true},
+}
+```
+
+Example identity data:
+```go
+{
+  "subscription_id": "00000000-0000-0000-0000-000000000000",
+  "resource_group_name": "exampleGroup",
+  "virtual_network_name": "exampleVNet",
+}
+```
+
+##### azurerm_role_definition
+Example schema:
+```go
+{
+  "scope_id": {string, required_for_import:true},
+  "role_definition_id": {string, required_for_import:true},
+}
+```
+
+Example identity data:
+```go
+{
+  "scope_id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myGroup",
+  "role_definition_id": "/subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Authorization/roleDefinitions/00000000-0000-0000-0000-000000000000",
+}
+```
+
+##### azurerm_role_assignment
+Example schema:
+```go
+{
+  "scope_id": {string, required_for_import:true},
+  "role_assignment_id": {string, required_for_import:true},
+}
+```
+
+Example identity data:
+```go
+{
+  "scope_id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/exampleGroup",
+  "role_assignment_id": "/providers/Microsoft.Authorization/roleAssignments/00000000-0000-0000-0000-000000000000",
+}
+```
+
+#### AzureAD
+
+##### azuread_group
+Example schema:
+```go
+{
+  "object_id": {string, required_for_import:true},
+}
+```
+
+Example identity data:
+```go
+{
+  object_id = "00000000-0000-0000-0000-000000000000"
+}
+```
+
+##### Azuread_group_member
+Example schema:
+```go
+{
+  "group_object_id": {string, required_for_import:true},
+  "member_object_id": {string, required_for_import:true},
+}
+```
+
+Example identity data:
+```go
+{
+  group_object_id = "00000000-0000-0000-0000-000000000000"
+  member_object_id = "11111111-1111-1111-1111-111111111111"
+}
+```
+##### azuread_application
+Example schema:
+```go
+{
+  "object_id": {string, required_for_import:true},
+}
+```
+
+Example identity data:
+```go
+{
+  object_id = "00000000-0000-0000-0000-000000000000"
+}
+```
+##### azuread_app_role_assignment
+Example schema:
+```go
+{
+  "principal_object_id": {string, required_for_import:true},
+  "app_role_id": {string, required_for_import:true},
+}
+```
+
+Example identity data:
+```go
+{
+  principal_object_id = "00000000-0000-0000-0000-000000000000"
+  app_role_id = "aaBBcDDeFG6h5JKLMN2PQrrssTTUUvWWxxxxxyyyzzz"
+}
+```
```

Comment:
I think we may have gotten wires crossed here, this is all information on how to retrieve an identity _schema_ from the CLI, whereas, the provider developer at this point is probably interested in "how/where do I [set identity data](https://github.com/hashicorp/terraform-provider-corner/blob/b06ba2acba9cc98cf58cda4a1ecd7239443c4c94/internal/framework5provider/identity_resource.go#L60-L63), how/where do I retrieve identity data (`Identity.Get`)"

Those would be in Go methods similar to the "Create" or "Read" sections of resource, but probably smaller:
- https://developer.hashicorp.com/terraform/plugin/framework/resources/create#define-create-method
- https://developer.hashicorp.com/terraform/plugin/framework/resources/read#define-read-method